app/pages/project/components/ProjectNavbar/components/ProjectTitle/ProjectTitle.jsx

```diff
@@ -87,17 +88,18 @@ function ProjectTitle({ launched, link, redirect, title, underReview }) {
           <span>
             {title}
             {redirect && <span>{' '}<i className="fa fa-external-link" /></span>}
-            {launched &&
-              <StyledCheckMarkWrapper
-                className="fa-stack"
-                aria-label={zooniverseApprovedTranslation}
-                title={zooniverseApprovedTranslation}
-              >
-                <i className="fa fa-circle fa-stack-2x" />
-                <StyledCheckMark className="fa fa-check fa-stack-1x" />
-              </StyledCheckMarkWrapper>}
           </span>
         </TitleComponent>
+        {launched &&
+          <StyledCheckMarkWrapper
+            className="fa-stack"
+            aria-label={zooniverseApprovedTranslation}
```

Comment:
`aria-label` is only announced on interactive elements. It won't work unless this component renders a link or button.

Reply:
VoiceOver announces this as "Zooniverse Approved, image" so 👍 from me. `aria-label` works on elements with `role="img"`.